src/test/scala/no/ndla/learningpathapi/service/ConverterServiceTest.scala

```diff
@@ -79,4 +82,17 @@ class ConverterServiceTest extends UnitSuite with UnitTestEnvironment {
       intercept[ValidationException] { service.asEmbedUrl(api.EmbedUrl("http://test.no/2/oembed/", Some("nb"), "test")) }.getMessage()
     }
   }
-}
\ No newline at end of file
+
+  test("asCoverPhoto converts an image id to CoverPhoto") {
+    val imageMeta = ImageMetaInformation("1",
+      "http://image-api.ndla-local/image-api/v1/images/1",
+      "http://image-api.ndla-local/image-api/raw/1.jpg",
+      1024, "something")
+    val expectedResult = CoverPhoto("http://proxy.ndla-local/image-api/raw/1.jpg", "http://proxy.ndla-local/image-api/v1/images/1")
```

Comment:
Nå føler jeg meg som han vanskelige igjen :)
Det er ikke her du ser for deg å gjøre url-konverteringen slik at lenkene blir "riktige"?